core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -146,7 +146,7 @@ migrate(Pause, Databases) ->
     io:format("removing depreciated databases...~n"),
     _  = remove_depreciated_databases(Databases),
 
-    kazoo_bindings:map(binding('migrate'), Accounts),
+    kazoo_bindings:map(binding('migrate'), [Accounts]),
```

Comment:
This would otherwise error in crossbar_maintenance receiving an account db instead of a list of account db.